lib/core/maven/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const { execSync } = require('child_process');
+const EXECUTABLE = 'java -jar '+__dirname+'/maven-model-helper.jar';
+
+function mergePoms(targetPath, sourcePath) {
+    execSync(EXECUTABLE + ' ' + targetPath + ' ' + sourcePath);
```

Comment:
Two things here:

1. perhaps a bit of validation? Throwing an exception when the JAR doesn't exist?

2. given the fact that I use promises everywhere else in the code perhaps we could do that here as well?

Like this `return exec(EXECUTABLE + ' ' + targetPath + ' ' + sourcePath)` and importing `child-process-promise`.